bosh_cpi/lib/cloud/external_cpi.rb

```diff
@@ -66,12 +67,15 @@ def ping(*arguments); invoke_cpi_method(__method__.to_s, *arguments); end
     private
 
     def invoke_cpi_method(method_name, *arguments)
+      context = {
```

Comment:
Makes sense. I removed the check for `context` in the openstack cpi: https://github.com/swisscom/bosh-openstack-cpi-release/commit/77970370f428447586925001414c4a84cb892d6d